### PR TITLE
Added text to erd in readme

### PR DESCRIPTION
**Pull request by Fahi28**
Added description to main README for the ERD.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ Amazon SES ensures reliable email delivery for user reports and notifications, w
 
 ![ERD Diagram](diagrams/erd.png)
 
+1. **3NF:** Schema has been put into 3NF to improve eliminate redundancy, maintain data integrity, improve query efficiency which is particularly important as all queries from the API will be regarding the ```earthquake``` table.
```
Comment (jiuliangut) [inline README.md:63]: do you mean 'to eliminate redundancy'?

**Review by jiuliangut — COMMENTED** [review]
looks good - just a small typo to go over but otherwise nice comprehemnsive descriptions :)

**Review by S1mpySloth — DISMISSED** [review]
Enriched the data modelling section with justifications, great work!

**Review by Jakub-Poskrop — DISMISSED** [review]
Other than the typo, it looks good.

**Review by S1mpySloth — APPROVED** [review]
Looks great, typo fixed!

**Review by Jakub-Poskrop — APPROVED** [review]
Looks good

**Pull request merged**